common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -236,6 +236,23 @@ def get_cohort_associated_content_group(self):
             return None
         return get_selected_option_text(self.q(css=self._bounded_selector(self.content_group_selector_css)))
 
+    def verify_cohort_content_group_selected(self, content_group=None):
+        """
+        Waits for the expected content_group (or none) to show as selected for
+        cohort associated content group.
+        """
+        if content_group:
+            self.wait_for(
+                lambda: unicode(
+                    self.q(css='select.input-cohort-group-association option:checked').text[0]
+                ) == content_group,
+                "Waiting to confirm cohort group has been selected.", timeout=10
```

Comment:
The message should be what was promised. In this case "Cohort group has been selected."
Also I'd leave the timeout as the default; it should return as soon as the promise is met anyhow.